src/library/scala/collection/mutable/ArrayBuilder.scala

```diff
@@ -61,8 +73,7 @@ sealed abstract class ArrayBuilder[T]
     val k = xs.knownSize
     if (k > 0) {
       ensureSize(this.size + k)
-      val actual = IterableOnce.copyElemsToArray(xs, elems, this.size)
-      if (actual != k) throw new IllegalStateException(s"Copied $actual of $k")
+      IterableOnce.copyElemsToArray(xs, elems, this.size)
```

Comment:
I'm assuming that you verified that `copyElemsToArray` will fail internally if it can't copy `knownSize` items, so the removed check is redundant?

Reply:
This was unintentional.  Must have been a stray `ctr + k` I unwittingly typed in the IDE and didn't catch at commit time.

Thank you for catching it, and for your patience with well meaning, but sloppy, first time contributors.  I'll exercise more caution in the next commit.

Reply:
History says the unused result was noticed by `-Wnonunit-statement`. The test example says "knownSize inconsistent with iterator".